client/my-sites/stats/activity-log-tasklist/index.jsx

```diff
@@ -0,0 +1,290 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { isEmpty, get, merge, each, omit, keyBy } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import ActivityLogTaskUpdate from './update';
+import alPluginsToUpdate from './plugins-to-update';
```

Comment:
"activity log" is already in the file name so it should be clear here without the awkward `al` prefix. Also wrappers are commonly named by what they provide "with"

```js
import WithUpdatablePlugins from './plugins-to-update';

export WithUpdatablePlugins( … );
```

Reply:
👍